pkg/docker/docker.go

```diff
@@ -542,6 +666,9 @@ func (d *stiDocker) GetScriptsURL(image string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if imageMetadata == nil {
```

Comment:
I could have swore I hit this during testing (should have left a comment), and at the time it appeared as if there was a path where the image was not found, but there was a no not found error.

That said, I walked down the code path, and it looks like it is returning a not found error in that case.

I'll remove it (for now), and it it shows up again when I re-run the test suite, I'll update here (and comment the code as to why we need the check).